pkg/model/awsmodel/iam.go

```diff
@@ -256,7 +256,7 @@ func (b *IAMModelBuilder) buildIAMRolePolicy(role iam.Subject, iamName string, i
 func (b *IAMModelBuilder) roleKey(role iam.Subject) (string, bool) {
 	serviceAccount, ok := role.ServiceAccount()
 	if ok {
-		return strings.ToLower(serviceAccount.Namespace + "-" + serviceAccount.Name), true
+		return strings.ToLower(strings.ReplaceAll(serviceAccount.Namespace+"-"+serviceAccount.Name, "*", "-")), true
```

Comment:
The `test---myserviceaccount_role_arn` terraform outputs below are a bit awkward. Maybe we can do this?

```suggestion
		return strings.ToLower(strings.ReplaceAll(serviceAccount.Namespace+"-"+serviceAccount.Name, "*", "wildcard")), true
```

That should result in `test-wildcard-myserviceaccount_role_arn` which is a bit easier to understand

Reply:
What bothers me with the wildcard literal is the potential collision with a namespace called wildcard.
Unfortunately, all substitution schemes seem to have collision issues though.

I’ll change the code as you suggest. We can always change that later if we come up with a better naming scheme.